contracts/evoting/controller/action.go

```diff
@@ -632,6 +558,86 @@ func (a *scenarioTestAction) Execute(ctx node.Context) error {
 	return nil
 }
 
+func setupSimpleElection(ctx node.Context,secret kyber.Scalar,proxyAddr1 string,serdecontext serde.Context, electionFac types.ElectionFactory,service ordering.Service) (error,string,types.Election,[]byte) {
```

Comment:
Here as well don't forget to add spaces. Moreover as the pipeline suggets, the error value is by convention the last argument returned.